src/Microsoft.TemplateEngine.Cli/New3Command.cs

```diff
@@ -424,7 +424,6 @@ private void FirstRun(IEngineEnvironmentSettings environmentSettings)
                 Reporter.Output.WriteLine(LocalizableStrings.GettingReady);
             }
             _callbacks.OnFirstRun?.Invoke(environmentSettings);
-            environmentSettings.SettingsLoader.Components.RegisterMany(typeof(New3Command).GetTypeInfo().Assembly.GetTypes());
```

Comment:
should we kill whole method instead? it's basically does nothing except printing "Getting ready" now...

Reply:
I'm wondering if having FirstRun makes any sense in general... If host need to keep track of that it should handle that themself... TemplateEngine has no need for this concept I believe...

Reply:
Lets leave changing this for another PR, its not effecting public API things...